rasa/importers/importer.py

```diff
@@ -161,9 +167,12 @@ def _importer_from_dict(
                 logging.warning(f"Importer '{module_path}' not found.")
                 return None
 
+        importer_config["training_type"] = training_type
```

Comment:
this line updates the `importer_config` dictionary, could it lead to unpredictable consequences, and would something like `importer_config = dict(training_type=training_type, **importer_config)` be more suitable? (or in 2 lines, copy() + update)